interfaces/smtp/v0/README.md

```diff
@@ -0,0 +1,64 @@
+# `smtp`
+
+## Overview
+
+This relation interface specification describes the expected behavior of charms integrating over the `smtp` Juju interface to provide or consume SMTP data.
+
+## Usage
+
+In most cases, this will be accomplished using the [smtp library](https://github.com/canonical/smtp-integrator-operator/blob/main/lib/charms/smtp_integrator/v0/smtp.py), although charm developers are free to provide alternative libraries as long as they fulfil the behavioural and schematic requirements described in this document.
+
+## Direction
+
+The `smtp` interface implements a provider/requirer pattern.
+The requirer is a charm that requires SMTP details to connect to an SMTP server, and the provider is a charm holding those details.
+
+```mermaid
+flowchart TD
+    Provider -- host, port, user, password_id, auth_type, transport_security, domain --> Requirer
```

Comment:
Is the `password_id` a Juju secret id? If yes, please specify it somewhere here.

Reply:
that also makes me think: how will either charm know it if the other end does or does not support secrets?

Reply:
The description of the different fields is in the schema definition. I don't see how that fits here

Reply:
Schema definition is also fine but even there it doesn't specifically say "Juju" secret.

Reply:
Updated in the schema definition